docs/DeveloperGuide.md

```diff
@@ -626,126 +511,104 @@ The application does not offer functionality for contacting tutees.
 
 <br>
 
-**UC11: Create a group**
+**UC11: View existing payment details of tutee**
 
 **MSS**
-1. User requests to create a group.
-2. System creates the group.
-
-**Extensions**
-* 1a. No group name is provided.
-    * 1a1. System shows an error message.
+1. User requests to list tutees. (UC04)
+2. System shows a list of tutees.
+3. User requests to view payment details of a specific tutee.
 
-      Use case resumes at step 1.
+   Use case ends.
 
-* 1b. The group name provided has already been used.
-    * 1b1. System shows an error message.
+   **Extensions**
 
-      Use case resumes at step 1.
+* 3a. The given index is invalid.
+    * 3a1. System shows an error message.
 
+      Use case resumes at step 2.
 <br>
 
-**UC12: Add a tutee to a group**
+**UC12: Add lesson fees to payment owed by tutee**
 
 **MSS**
-1. User requests to list tutees.
+1. User requests to list tutees. (UC04)
 2. System shows a list of tutees.
-3. User requests to add a specific tutee in the list to a group.
-4. System adds the tutee to the group.
+3. User requests to view payment details of tutee. (UC11)
+4. User selects lesson and adds fees of lesson to existing fees.
+5. System shows new updated payment details.
 
    Use case ends.
 
-**Extensions**
-* 2a. The list is empty.
-
-  Use case ends.
+   **Extensions**
```

Comment:
Perhaps you may want to add another Extension where the user wants to add a lesson fee that will cause the payment amount to exceed the maximum allowed of 10,000?